src/main/java/io/arlas/server/ogc/csw/utils/ElementSetName.java

```diff
@@ -0,0 +1,4 @@
+package io.arlas.server.ogc.csw.utils;
+
+public enum ElementSetName {
```

Comment:
This class doesn't seem to be directly linked to this commit.

Reply:
even if it is modified in the next commit.

Reply:
Fixed